airbyte-cdk/python/airbyte_cdk/sources/lcc/requesters/paginators/interpolated_paginator.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, List, Mapping, Optional
+
+import requests
+from airbyte_cdk.sources.lcc.interpolation.interpolated_mapping import InterpolatedMapping
+from airbyte_cdk.sources.lcc.interpolation.jinja import JinjaInterpolation
+from airbyte_cdk.sources.lcc.requesters.paginators.paginator import Paginator
+
+
+class InterpolatedPaginator(Paginator):
+    def __init__(self, next_page_token: Mapping[str, str], config):
```

Comment:
Good question!
Yes, this would be a next_page_token template. I'll rename
Here's a concrete example for sendgrid: https://github.com/airbytehq/airbyte/pull/12855/files#diff-5b8bb05631d1c4696a82988e2fc4d9d4d87061ad89050db55c66e8b68cc0314cR42